test/e2e/framework/.import-restrictions

```diff
@@ -190,7 +190,7 @@
 				"k8s.io/kubernetes/pkg/scheduler/internal/parallelize",
 				"k8s.io/kubernetes/pkg/scheduler/listers",
 				"k8s.io/kubernetes/pkg/scheduler/metrics",
-				"k8s.io/kubernetes/pkg/scheduler/nodeinfo",
+				"k8s.io/kubernetes/pkg/scheduler/types",

```

Comment:
It looks like you can't simply remove `nodeinfo` due to the dependency chain: `e2e/framework` -> `kubemark` -> `kubelet` -> `scheduler`

Reply:
yes, I just added it back.